### DISCUSSION TITLE
Inquiry of migrating client implementation to typescript

### DISCUSSION THREAD

**Discussion by andersakerberg** [tewtal/SMZ3Randomizer#150] [Development]
Hello! I created a fork fo this repository and typescriptified the React APP hosted on https://alpha.samus.link && https://samus.link , the changes are discoverable at https://github.com/tewtal/SMZ3Randomizer/compare/master...andersakerberg:master , I am a developer since 15+ years and would love to contribute in whichever way I can, please give a shout if you feel like you could use some support! 

I did not go as far as to test the app but it runs atleast :) Unfourtunately mac osx only has support for netcore 5

Love the work <3

**Comment by RebelusQuo** [comment]
Given the size, and relatively low complexity of the current implementation, I don't see any reason to change to typescript. From my quick read this seems like a 1 to 1 translation to .ts files, correct?

**Comment by andersakerberg** [comment]
Its a bit more then that, intially yes, but then its about typing up everything and securing new version from not breaking the site. To acheive that i would need to align the typings with someone who has knowledge of this. 

But anywhere else I can be of use?

**Comment by RebelusQuo** [comment]
We're wanting to get a 11.2 release done as soon as possible with keysanity, and therefore we're touching as little as possible. After that release I intend to work on the javascript structure. Unit and system tests would be nice. We can revisit later on.
